packages/App/src/balance/wallet/Categories.tsx

```diff
@@ -1,7 +1,15 @@
-import React from 'react';
-import {StyleSheet, View, Text, Image} from 'react-native';
+import React, {FC} from 'react';
+import {StyleSheet, View, Text, Image, ImageProps} from 'react-native';
 
-const Categories = ({category, amount, date, type, icon}: any) => {
+interface Props {
+  category: string;
+  amount: number;
```

Comment:
This might be useful to extract a generic type for money amount into a separate type.
`type AmountInCents = number`
This will make it easier to you to always remember that `amount` includes cents